testdata/bundles/registry-v1/build-push-e2e-bundle.sh

```diff
@@ -0,0 +1,81 @@
+#! /bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+help="
+build-push-e2e-bundle.sh is a script to build and push the e2e bundle image using kaniko.
+Usage:
+  build-push-e2e-bundle.sh [NAMESPACE] [TAG] [BUNDLE_NAME] [BUNDLE_DIR]
+
+Argument Descriptions:
+  - NAMESPACE is the namespace the kaniko Job should be created in
+  - TAG is the full tag used to build and push the catalog image
+"
+
+if [[ "$#" -ne 4 ]]; then
+  echo "Illegal number of arguments passed"
+  echo "${help}"
+  exit 1
+fi
+
+
+namespace=$1
+tag=$2
+bundle_name=$3
+package_name=$4
+bundle_dir="testdata/bundles/registry-v1/${package_name}"
+
+echo "${namespace}" "${tag}"
+
+kubectl create configmap -n "${namespace}" --from-file="${bundle_dir}/Dockerfile" operator-controller-e2e-${bundle_name}.root
+
+tgz="${bundle_dir}/manifests.tgz"
+tar czf "${tgz}" -C "${bundle_dir}/" manifests metadata
+kubectl create configmap -n "${namespace}" --from-file="${tgz}" operator-controller-${bundle_name}.manifests
+rm "${tgz}"
+
+kubectl apply -f - << EOF
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "kaniko-${bundle_name}"
```

Comment:
Because we have dots in bundle names, this results in:

```
Warning: metadata.name: this is used in Pod names and hostnames, which can result in surprising behavior; a DNS label is recommended: [must not contain dots]
```

It is just a warning, but a bit annoying.

Reply:
Fixed.